engine/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -157,9 +159,12 @@ class WorkflowActorSpec extends CromwellTestKitSpec with WorkflowDescriptorBuild
     "copy workflow logs in the event of MaterializeWorkflowDescriptorFailureResponse" in {
       val actor = createWorkflowActor(MaterializingWorkflowDescriptorState)
       deathwatch watch actor
+
+      copyWorkflowLogsProbe.expectNoMsg(AwaitAlmostNothing)
```

Comment:
This one makes sure that the `copyWorkflowLogsProbe` doesn't receive any messages before the materialization happens.
Line 167 makes sure the finalization actor never receives any messages throughout this test